ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelcontainer/v1/panelcontainer/README.md

```diff
@@ -0,0 +1,76 @@
+<!--
+Copyright 2022 Adobe
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+Panel Container (v1)
+====
+Panel Container component written in HTL.
+
+## Features
+
+* Configurable layout type.
+* Configurable background image and background color:
+    * Background images and colors can be enabled through policy configuration.
+    * Color swatches for background color can be defined through policy configuration.
+    * Background color can be restricted to only allow swatches through policy configuration.
+* Configurable HTML ID attribute.
+* Allowed components can be configured through policy configuration.
+* Style System support.
```

Comment:
I think this needs to be fixed

Reply:
Above is inherited from sites container. For now I have added:
Apply a `data-cmp-is="adaptiveFormPanel"` attribute to the wrapper block to enable initialization of the JavaScript component.